### PR TITLE
fix: btn default css

### PR DESCRIPTION
**Pull request by Harris-YWang**
## Description

Further fix btn default css

Have been fully tested with `adslot-grid` and `direct-web`. Should be okay after merging.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by glenkurniawan-adslot — APPROVED** [review]

**Review by pphminions — APPROVED** [review]
LGTM 🦋

File: src/components/Button/styles.scss [hunk 1]
```diff
@@ -169,14 +174,21 @@
...
+    &:active,
+    &:hover,
+    &:hover:active,
     &:focus {
       background-color: transparent;
       border-color: transparent;
       color: $color-blue-dark;
       box-shadow: none;
-      text-decoration: underline;
+      text-decoration: none;
```
Comment (superpower1) [inline src/components/Button/styles.scss:185]: Wondering why you remove the text decoration for link button.

**Review by adityasabnis — APPROVED** [review]

File: src/components/Button/styles.scss (hunk 1 above)
Comment (Harris-YWang) [reply src/components/Button/styles.scss:185]: just remove it after users clicking the button. When users hovering on it, `text-decoration` is still there, please have a look at the code below

**Review by superpower1 — APPROVED** [review]
Buddy Tested, LGTM.

**Comment by pphminions** [comment]
merging 🚀

**Pull request merged**